readthedocs/api/mixins.py

```diff
@@ -55,3 +61,35 @@ def _get_cache_tags(self):
         if project and self.project_cache_tag:
             tags.append(get_cache_tag(project.slug, self.project_cache_tag))
         return tags
+
+
+class EmbedAPIMixin:
```

Comment:
This class should have a docstring, probably explaining why we're caching these various things?